cmake/modules/boards.cmake

```diff
@@ -144,29 +235,12 @@ if(DEFINED BOARD_REVISION)
   string(REPLACE "." "_" BOARD_REVISION_STRING ${BOARD_REVISION})
 endif()
 
-message(STATUS "${board_message}")
-
-# Prepare boards usage command printing.
-# This command prints all boards in the system in the following cases:
-# - User specifies an invalid BOARD
-# - User invokes '<build-command> boards' target
-list(TRANSFORM ARCH_ROOT PREPEND "--arch-root=" OUTPUT_VARIABLE arch_root_args)
-list(TRANSFORM BOARD_ROOT PREPEND "--board-root=" OUTPUT_VARIABLE board_root_args)
-
-set(list_boards_commands
-    COMMAND ${PYTHON_EXECUTABLE} ${ZEPHYR_BASE}/scripts/list_boards.py
-            ${arch_root_args} ${board_root_args} --arch-root=${ZEPHYR_BASE}
-)
-
-if(NOT BOARD_DIR)
-  message("No board named '${BOARD}' found.\n\n"
-          "Please choose one of the following boards:\n"
-  )
-  execute_process(${list_boards_commands})
-  unset(CACHED_BOARD CACHE)
-  message(FATAL_ERROR "Invalid BOARD; see above.")
+if(DEFINED BOARD_IDENTIFIER)
+  set(board_message "${board_message}, variant: ${BOARD_IDENTIFIER}")
```

Comment:
would think the `/` should be removed from this output

Reply:
fixed